src/components/ViewOrder.jsx

```diff
@@ -1,12 +1,12 @@
 import React, { Component } from 'react'
-import {getOrder} from '../modules/orderData'
-import {Item, Container, Header} from 'semantic-ui-react'
+import {getOrder, closeOrder} from '../modules/orderHelper'
```

Comment:
why is it changed from /orderData to /orderHelper ?

Reply:
we decided to move all order-related functions to one single file

Reply:
Great idea!